generate.sh

```diff
@@ -15,6 +15,10 @@ do
  esac
 done
 
+function print_log {
+  echo -e $1 | tee -a ${LOGFILE}
```

Comment:
Do something like `if [ -n "$LOGFILE" ]` and in the else section just do a simple `echo`. And define LOGFILE in the webUI section of the script. Thus from travis All Logs would be printed properly and from the webUI we would have a concise output.